### PR TITLE
post: what's new in Angular 2.1

### PR DESCRIPTION
**Pull request by cexbrayat**
Small blog post on the minor release of Angular.
Might be better to run this one before JB's one, as it is more time sensitive :)

I plan to write a blog post every month for the new releases,
probably a great way to advertise our ebook ;)

### CODE REVIEW

File: _posts/2016-10-03-what-is-new-angular-2.1.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+layout: post
+title: What's new in Angular 2.1?
+author: cexbrayat
+tags: ["angular2"]
+description: "Angular 2.1 is out. What new features are included?"
+---
+
+Angular is moving fast, and we already have a minor release!
+
+If you didn't heard, Angular is planning to have patch updates every week,
```
Comment (jnizet) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: If you haven't heard

File: _posts/2016-10-03-what-is-new-angular-2.1.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+We'll ignore the tons of bug fixes, perf improvements and changes to ngUpgrade,
+to only focus on the new features.
+
+This blog post contains the changes from Angular 2.0.x releases
+and Angular 2.1.0 various betas and RCs.
+
+## Router, modules, lazy-loading and pre-loading
+
+The router comes with a really amazing feature
+that allows to lazy-load parts of your application.
```
Comment (jnizet) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: allowing to

File: _posts/2016-10-03-what-is-new-angular-2.1.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Instead of shipping a big bundle of all your app when your user lands on your home page,
```
Comment (jnizet) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: containing the whole application

File: _posts/2016-10-03-what-is-new-angular-2.1.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+          ])
+        ])
+      ]
+    })
+    export class AnimatedComponent {
+
+With this animation, the component will slowly "fade in".
+
+That's all for this small release.
+Of course our [ebook](https://books.ninja-squad.com) and [Pro Pack](https://angular2-exercises.ninja-squad.com/) are now up-to-date with the changes introduced,
```
Comment (jnizet) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: Are they?

File: _posts/2016-10-03-what-is-new-angular-2.1.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+(if the user is not an Administrator, maybe you can safely ignore the `AdminModule` for example).
+
+## Animations
+
+Angular also comes with a great support for animations,
+with a custom DSL to define them.
+Angular 2.1.0 comes with two handy aliases `:enter` and `:leave`,
+to define animations that should run when the component is created or deleted.
+
+    @Component({
```
Comment (jnizet) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: A link to a plnkr would be nice and easy.

File: _posts/2016-10-03-what-is-new-angular-2.1.md (hunk 4 above)
Comment (cexbrayat) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: Well, we don't talk about animations or lazy-loading yet, but we are up to date with the rest...

File: _posts/2016-10-03-what-is-new-angular-2.1.md (hunk 5 above)
Comment (jnizet) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: We could also make an animated gif out of it and add it into the post.

File: _posts/2016-10-03-what-is-new-angular-2.1.md (hunk 4 above)
Comment (jnizet) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: Hmm, I'm not comfortable with that part then. By reading that, I have the feeling that these two changes are the mort important ones, and that the book contains them.

Comment (cexbrayat) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: OK, I amended the conclusion

File: _posts/2016-10-03-what-is-new-angular-2.1.md (hunk 5 above)
Comment (cexbrayat) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: Probablement, mais j'ai la flemme :)

Comment (jnizet) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: http://plnkr.co/edit/3Y0ODbdFiCkh6XGrxFRA?p=preview

Comment (cexbrayat) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: 👌

File: _posts/2016-10-03-what-is-new-angular-2.1.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: What's new in Angular 2.1?
+author: cexbrayat
+tags: ["angular2"]
```
Comment (clacote) [inline _posts/2016-10-03-what-is-new-angular-2.1.md]: angular2 -> Angular 2 (as in http://blog.ninja-squad.com/tags.html#Angular 2-ref) (some posts do use "angular2" tag: I will fix in a coming PR)

**Pull request merged**